TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -58,13 +58,14 @@ class RouteOptionsViewController: UIViewController, UITableViewDelegate, UITable
 
     // MARK: Reachability vars
 
-    let reachability: Reachability? = Reachability(hostname: Network.source)
+    let reachability: Reachability? = Reachability(hostname: Network.ipAddress)
+    
     var banner: StatusBarNotificationBanner = {
-        let banner = StatusBarNotificationBanner(title: "No Internet Connection", style: .danger)
+        let banner = StatusBarNotificationBanner(title: "No internet connection. Retrying...", style: .danger)
```

Comment:
Does it retry tho? i remember bef austin had a retry button on his view so i assumed reachability doesn't retry the network connection. Maybe test to make sure that if get no internet connection banner but then you connect to internet, the reachability banner disappears

Reply:
Mihir suggested it to show we're always trying. Theoretically it should go away once it changes

Reply:
okay if does go away when we reconnect to internet then it should be okay. don't want to mislead user